lib/membrane/rtcp/receiver.ex

```diff
@@ -78,12 +75,7 @@ defmodule Membrane.RTCP.Receiver do
 
   @impl true
   def handle_event(:input, %RTCPEvent{rtcp: %SenderReportPacket{} = rtcp} = event, _ctx, state) do
-    Membrane.TelemetryMetrics.execute(
-      @sender_report_received_telemetry_event,
-      %{},
-      %{},
-      state.telemetry_label
-    )
+    emit_sender_report_sent_telemetry_event(state)
```

Comment:
1. This function name is invalid (it sends `@sender_report_reeceived_telemetry_event`)
2. I rather had this in mind:
```suggestion
    emit_telemetry_event(@sender_report_received, state)
```
This way the private function could extract the common part. Now you still have duplication, although grouped. However, I won't insist on that change